reconcile/vault_replication.py

```diff
@@ -0,0 +1,187 @@
+import logging
```

Comment:
Since this is an entirely new integration, can you please create tests for it?

Reply:
Added some tests, will add more later today :)